### PR TITLE
Fix build and test breakages

### PR DESCRIPTION
**Pull request by jeremyje**
Fixing the build, vet, fmt errors in open-match.

### CODE REVIEW

File: internal/statestorage/redis/ignorelist/ignorelist_test.go [hunk 1]
```diff
@@ -6,12 +6,14 @@ import (
...
+func TestCreate(t *testing.T) {
+	pids := []string{"test1", "test2", " test3"}
 	ilid := "testil"
 	redisConn := redigomock.NewConn()
+
+	redisConn.GenericCommand("ZADD").Expect("ok")
 	err := Create(redisConn, ilid, pids)
 	if err != nil {
-		panic(err.Error())
+		t.Error(err)
```
Comment (jkowalski) [inline internal/statestorage/redis/ignorelist/ignorelist_test.go]: t.Fatal() ?

File: test/cmd/frontendclient/main.go [hunk 2]
```diff
@@ -194,7 +194,7 @@ func waitForResults(resultsChan chan frontend.Player, stream frontend.Frontend_G
 				break
 			}
 			log.Println("Error encountered")
-			log.Println("Error reading stream for GetAssignments(_) = _, %v", err)
+			log.Printf("Error reading stream for GetAssignments(_) = _, %v\n", err)
```
Comment (jkowalski) [inline test/cmd/frontendclient/main.go]: nit: log.Printf() does not need a trailing newline

File: test/cmd/frontendclient/main.go [hunk 3]
```diff
@@ -194,7 +194,7 @@ func waitForResults(resultsChan chan frontend.Player, stream frontend.Frontend_G
 				break
 			}
 			log.Println("Error encountered")
-			log.Println("Error reading stream for GetAssignments(_) = _, %v", err)
+			log.Printf("Error reading stream for GetAssignments(_) = _, %v", err)
```
Comment (calebatwd) [inline test/cmd/frontendclient/main.go]: nit/curious. I haven't seen placeholders like the "*_*" before to visualize the input and right-side? Is that a standard pattern/does it work with the logging library in some interesting way? (I realize it's carry-over from the previous code... just curious :) )

Comment (jeremyje) [reply test/cmd/frontendclient/main.go]: Not sure probably not worth looking into just because this code is probably going to die.

**Pull request merged**